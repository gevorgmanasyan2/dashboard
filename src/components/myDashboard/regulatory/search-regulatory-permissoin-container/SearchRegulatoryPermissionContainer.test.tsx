/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import SearchRegulatoryApprovalContainer from './SearchRegulatoryPermissionContainer'

it('should render SearchRegulatoryApprovalContainer component', () => {
  expect(shallow(<SearchRegulatoryApprovalContainer />)).toMatchSnapshot()
})
