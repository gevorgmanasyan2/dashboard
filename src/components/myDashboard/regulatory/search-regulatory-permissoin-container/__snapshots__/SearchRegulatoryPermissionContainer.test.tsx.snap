// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchRegulatoryApprovalContainer component 1`] = `
<article
  className="searchRegulatory__content"
>
  <header
    className="searchRegulatory__content__header"
  >
    <Title
      titleText="Regulatory Permission"
    >
       
    </Title>
    <div
      className="searchRegulatory__content__header--icon"
    >
      <ForwardRef(PlusOutlined) />
    </div>
  </header>
  <div
    className="searchRegulatory__content--searchInput"
  >
    <ShareableInput
      placeholder="Search"
    />
  </div>
  <div
    className="searchRegulatory__content--regulatory"
  >
    <Connect(Regulatory) />
  </div>
  <div
    className="searchRegulatory__content--pagination"
  >
    <Pagination
      defaultCurrent={1}
      total={50}
    />
  </div>
</article>
`;
